text/0089-metrics-views-api.md

```diff
@@ -0,0 +1,205 @@
+# Metrics View API
+
+This OTEP addresses the [Future Work: Configurable Aggregations / View API](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-metrics.md#future-work-configurable-aggregations--view-api) section of [api-metrics.md](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-metrics.md).
+It proposes a specification for a _view API_ that allows users to configure aggregations for individual metric instruments.
```

Comment:
Can we clarify who "users" are in this context of this OTEP? Are we talking about instrumentation authors, or application developers/operators, or authors of exporters?

Reply:
Changed to "application owners and administrators".

It's probably also worth pointing out here that integration authors might want to ship a set of views too, like https://www.javadoc.io/doc/io.opencensus/opencensus-contrib-grpc-metrics/0.17.0/io/opencensus/contrib/grpc/metrics/RpcViewConstants.html.